packages/bp/e2e/studio/cms.test.ts

```diff
@@ -30,63 +36,71 @@ describe('Studio - CMS', () => {
 
     await clickOn('#btn-filter-builtin_text')
     await expectStudioApiCallSuccess('cms/builtin_text/elements', 'POST')
+
     const after = await getElementCount()
     expect(after).toBeLessThan(before)
   })
 
   it('Create an image element', async () => {
     const before = await getElementCount(true)
+
     await page.hover('#btn-filter-builtin_image')
     await clickOn('#btn-list-create-builtin_image')
     await uploadFile('input[type="file"]', path.join(__dirname, '../assets/alien.png'))
     await expectStudioApiCallSuccess('media', 'POST')
+
     await clickOn('.DraftEditor-root')
     await page.keyboard.type('I am a martian')
     await clickOn('button[type="submit"]')
     await expectStudioApiCallSuccess('cms/builtin_image/elements', 'POST')
+
     const after = await getElementCount(true)
     expect(after).toBeGreaterThan(before)
   })
 
   it('Create a file element', async () => {
     const before = await getElementCount(true)
+
     await page.hover('#btn-filter-builtin_file')
     await clickOn('#btn-list-create-builtin_file')
     await uploadFile('input[type="file"]', path.join(__dirname, '../assets/README.pdf'))
     await expectStudioApiCallSuccess('media', 'POST')
+
     await clickOn('.style__textarea___2P8hT')
     await page.keyboard.type('Botpress README')
     await clickOn('button[type="submit"]')
     await expectStudioApiCallSuccess('cms/builtin_file/elements', 'POST')
+
     const after = await getElementCount(true)
     expect(after).toBeGreaterThan(before)
   })
 
-  // it('Create text element', async () => {
+  it('Create text element', async () => {
+    const before = await getElementCount(true)
 
-  //   await page.keyboard.press('Tab')
-  //   await page.keyboard.type('hey!')
-  //   await clickOn('button[type="submit"]')
+    await page.hover('#btn-filter-builtin_text')
+    await clickOn('#btn-list-create-builtin_text')
 
-  //   await expectBotApiCallSuccess('content/builtin_text/element', 'POST')
-  //   await page.waitFor(500) // Ensure the element is created and the list is reloaded
-  //   const after = await getElementCount()
+    await clickOn('.style__textarea___2P8hT')
```

Comment:
I'm not sure it's a good idea to base anything on a randomly generated ID on the style, is there an alternative?

Reply:
Good point. I forgot to add ids to those components. I'll do those changes!